rules/stylistic-issues.js

```diff
@@ -0,0 +1,70 @@
+		"indent": [ 2, "tab", { "SwitchCase": 1 } ], // JSCS specify tab or space width for your code (fixable)
+		"jsx-quotes": [ 2, "prefer-double" ], // specify whether double or single quotes should be used in JSX attributes
+		"key-spacing": [ 2, { "beforeColon": false, "afterColon": true } ], // JSCS enforce spacing between keys and values in object literal properties
+		"linebreak-style": [ 2, "unix" ], // JSCS disallow mixed 'LF' and 'CRLF' as linebreaks
+		"lines-around-comment": 0, // JSCS enforce empty lines around comments
+		"max-depth": [ 2, 5 ], // specify the maximum depth that blocks can be nested
+		"max-len": 0, // specify the maximum length of a line in your program
+		"max-nested-callbacks": [ 2, 3 ], // specify the maximum depth callbacks can be nested
+		"max-params": [ 2, 5 ], // limits the number of parameters that can be used in the function declaration.
+		"max-statements": [ 2, 25 ], // specify the maximum number of statement allowed in a function
+		"new-cap": 2, // JSCS require a capital letter for constructors
+		"new-parens": 2, // disallow the omission of parentheses when invoking a constructor with no arguments
+		"newline-after-var": [ 1, "always" ], // require or disallow an empty newline after variable declarations
+		"no-array-constructor": 2, // disallow use of the Array constructor
+		"no-bitwise": 0, // JSHINT disallow use of bitwise operators
+		"no-continue": 2, // disallow use of the continue statement
+		"no-inline-comments": 0, // disallow comments inline after code
+		"no-lonely-if": 2, // disallow if as the only statement in an else block
+		"no-mixed-spaces-and-tabs": 2, // JSCS disallow mixed spaces and tabs for indentation
+		"no-multiple-empty-lines": [ 2, { max: 2 } ], // JSCS disallow multiple empty lines
+		"no-negated-condition": 2, // disallow negated conditions
+		"no-nested-ternary": 2, // disallow nested ternary expressions
+		"no-new-object": 2, // disallow the use of the Object constructor
+		"no-plusplus": 0, // JSHINT disallow use of unary operators, ++ and --
+		"no-restricted-syntax": [ 2, "WithStatement" ], // disallow use of certain syntax in code
+		"no-spaced-func": 2, // JSCS disallow space between function identifier and application (fixable)
+		"no-ternary": 0, // disallow the use of ternary operators
+		"no-trailing-spaces": 2, // JSCS disallow trailing whitespace at the end of lines (fixable)
+		"no-underscore-dangle": 0, // disallow dangling underscores in identifiers
+		"no-unneeded-ternary": 2, // disallow the use of ternary operators when a simpler alternative exists
+		"object-curly-spacing": [ 2, "always" ], // require or disallow padding inside curly braces (fixable)
+		"one-var": [ 2, { "initialized": "never", "uninitialized": "always" } ], // JSCS require or disallow one variable declaration per function
+		"operator-assignment": [ 1, "always" ], // require assignment operator shorthand where possible or prohibit it entirely
```

Comment:
Discuss, do we want this to be ignored (0) or to be an error (2)?

http://eslint.org/docs/rules/operator-assignment.html